src/components/CoSign/CosignDone.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Anchor from 'components/Anchor';
+import Body from 'components/Body';
+import Card from 'components/Card';
+import FAIcon from 'components/FAIcon';
+
+const CosignDone = ({reportDownloadUrl}) => {
+  return (
+    <Card
+      title={
+        <FormattedMessage
+          description="On succesful completion title"
+          defaultMessage="Co-sign confirmation"
+        />
+      }
+    >
+      {/* TODO Make text configurable */}
+      <Body component="div" modifiers={['wysiwyg']}>
+        <FormattedMessage
+          description="Co-sign confirmation page body"
+          defaultMessage="Thank you for co-signing this submission."
+        />
+      </Body>
+
+      <Body>
+        <FAIcon icon="download" aria-hidden="true" modifiers={['inline']} />
```

Comment:
the `main` branch now has `aria-hidden={true}` as default

```suggestion
        <FAIcon icon="download" modifiers={['inline']} />
```